WorkflowSwiftUI/Tests/PreferredContentSizeTests.swift

```diff
@@ -6,7 +6,12 @@ import WorkflowSwiftUI
 import XCTest
 
 final class PreferredContentSizeTests: XCTestCase {
-    func test_preferredContentSize() {
+    func test_preferredContentSize() throws {
+        guard #unavailable(iOS 18.0) else {
+            try XCTSkipIf(true, "Intrinsic content size does not update unless hosted in an actual app on iOS 18")
+            return
+        }
```

Comment:
not sure what a better solution to this problem is (iOS 18 runtime seemingly doesn't invalidate/update VCs in the same way it used to)... use app hosted unit tests?

Reply:
sounds good, but let's do as a separate PR then (unless it requires very little code change)

Reply:
this 'workaround' didn't actually seem to work around the problem for some reason, so i've just disabled the test currently